### PR TITLE
Bug/xot 712 remove links in search results

### PR DESCRIPTION
**Pull request by WillTaylor22**

### CODE REVIEW

File: core/serializers.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+def parse_search_results(content):
+    results = [hit['_source'] for hit in content['hits']['hits']]
+
+    # This is to remove HTML tags and markdown received from CMS
+    # or any other source.
+    #
+    # It first converts the markdown received into HTML
+    # Then uses a regex expression to remove HTML tags
+    # It also removes unneccessary \n added by markdown
+    tag_remover = re.compile(r'<[^>]+>')
```
Comment (richtier) [inline core/serializers.py]: using regex with html is notoriously flakey.

fortunately, we have BeautifulSoup installed already :
```
from bs4 import BeautifulSoup

html = markdown2.markdown(result['content'])
text = ''.join(BeautifulSoup(html).findAll(text=True))
```

File: makefile [hunk 2]
```diff
@@ -88,11 +88,10 @@ DEBUG_SET_ENV_VARS := \
...
 	export COMMUNITY_ENQUIRIES_AGENT_EMAIL_ADDRESS=community@example.com \
-	export ACTIVITY_STREAM_API_URL=http://localhost:8080/v1/objects; \
-	export ACTIVITY_STREAM_API_SECRET_KEY=incoming-some-secret-3; \
-	export ACTIVITY_STREAM_API_ACCESS_KEY=incoming-some-id-3; \
-	export ACTIVITY_STREAM_API_IP_WHITELIST=1.2.3.4,2.3.4.5
-
+	export ACTIVITY_STREAM_API_ACCESS_KEY=a1174d82d158b3d1ffb7; \
+	export ACTIVITY_STREAM_API_IP_WHITELIST=1.2.3.4; \
+	export ACTIVITY_STREAM_API_SECRET_KEY=7483eb4a44460a8c570928068568436c368cbc07 ;\
+	export ACTIVITY_STREAM_API_URL=https://activity-stream-staging.london.cloudapps.digital/v1/objects
```
Comment (richtier) [inline makefile]: the secrets are not compromised. please change them as a matter of urgency. it's not enough to just push with force to remove this from history - bots actively look for things like this and store it

Comment (nusnewob) [reply makefile]: Can someone force push to overwrite any domains?

Comment (WillTaylor22) [reply makefile]: This has been changed

Comment (richtier) [reply makefile]: I think the ask it to git commit ammend and then force push to hide the domains from casual observers.

Comment (WillTaylor22) [reply makefile]: Done... but how do we delete this conversation here!

File: README.md [hunk 3]
```diff
@@ -66,6 +66,32 @@ EXPORTING_OPPORTUNITIES_SEARCH_URL=debug
...
+CONTACT_INTERNATIONAL_AGENT_EMAIL_ADDRESS=debug
+CONTACT_ENQUIRIES_AGENT_EMAIL_ADDRESS=debug
+EXPORTING_OPPORTUNITIES_API_BASE_URL=debug
+EXPORTING_OPPORTUNITIES_API_SECRET=debug
+EXPORTING_OPPORTUNITIES_SEARCH_URL=debug
+GET_ADDRESS_API_KEY=debug
+ACTIVITY_STREAM_API_ACCESS_KEY=123
+ACTIVITY_STREAM_API_IP_WHITELIST=1.2.3.4
+ACTIVITY_STREAM_API_SECRET_KEY=123
+ACTIVITY_STREAM_API_URL=http://localhost:8080/v1/objects
```
Comment (richtier) [inline README.md]: another PR beat you to it :) Please add the `ACTIVITY_STREAM_API_*` entries to the list that starts at line 54

File: makefile [hunk 4]
```diff
@@ -123,9 +119,10 @@ TEST_SET_ENV_VARS := \
 	export CONTACT_EXPORTING_AGENT_SUBJECT=exporting-subject; \
 	export COMPANIES_HOUSE_API_KEY=debug; \
 	export CONTACT_ENQUIRIES_AGENT_EMAIL_ADDRESS=domesitc-enquiries@example.com; \
-	export ACTIVITY_STREAM_API_URL=http://localhost:8080/v1/objects; \
-	export ACTIVITY_STREAM_API_SECRET_KEY=incoming-some-secret-3; \
-	export ACTIVITY_STREAM_API_ACCESS_KEY=incoming-some-id-3
+  export ACTIVITY_STREAM_API_URL=https://www.nonworkingdomaintoraiseconnectionerror.com; \
+  export ACTIVITY_STREAM_API_SECRET_KEY=debug; \
+  export ACTIVITY_STREAM_API_ACCESS_KEY=debug; \
+  export ACTIVITY_STREAM_API_IP_WHITELIST=debug
```
Comment (richtier) [inline makefile]: funky indentation :)

Comment (WillTaylor22) [reply makefile]: Thanks

File: README.md (hunk 3 above)
Comment (WillTaylor22) [reply README.md]: Done

**Pull request merged**